packages/pds/src/index.ts

```diff
@@ -242,14 +248,39 @@ export class PDS {
       algos,
     })
 
-    let server = createServer({
+    const xrpcOpts: XrpcServerOptions = {
       validateResponse: config.debugMode,
       payload: {
         jsonLimit: 100 * 1024, // 100kb
         textLimit: 100 * 1024, // 100kb
         blobLimit: 5 * 1024 * 1024, // 5mb
       },
-    })
+    }
+    if (config.rateLimitsEnabled) {
+      xrpcOpts['rateLimits'] = {
+        creator: (opts: RateLimiterOpts) =>
+          RateLimiter.memory({
+            bypassSecret: config.rateLimitBypassKey,
+            ...opts,
+          }),
+        global: [
+          {
+            name: 'global-ip',
+            durationMs: 5 * MINUTE,
+            points: 1000,
```

Comment:
Will `OPTIONS` requests get counted against this limit?

Reply:
no they shouldn't be - the rate limit consumption occurs in xrpc-server directly before the handler